src/components/Card/WorkoutCard.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+function WorkoutCard(props) {
```

Comment:
whats the props you are receiving?
this should look like this
```suggestion
function WorkoutCard({title, id, image, duration, type}) {
```